app/helpers/movies_helper.rb

```diff
@@ -68,6 +68,24 @@ def display_pay_model_icon(movie_pay_model)
     end
   end
 
+  def firstify_number(number)
```

Comment:
I could not find a helper method to convert a `1` to `1st`, so I made one.

Reply:
😎 real nice!